packages/ai/src/LLMStep.ts

```diff
@@ -69,14 +73,16 @@ export class LLMStepTemplate<const Shape extends StepShape = StepShape> {
 
   instantiate(
     workflow: Workflow,
-    inputs: Inputs<Shape>
+    inputs: Inputs<Shape>,
+    retryingStep?: LLMStepInstance<Shape> | undefined
   ): LLMStepInstance<Shape> {
-    return new LLMStepInstance(this, workflow, inputs);
+    return new LLMStepInstance(this, workflow, inputs, retryingStep);
   }
 }
 
 export class LLMStepInstance<const Shape extends StepShape = StepShape> {
   public id: string;
+  public inputs: Inputs<Shape>;
```

Comment:
Inputs were already exposed via `getInputs()`, btw. But it's not important

Reply:
It's not very helpful in this particular case, but sometimes you want to make the field mutable internally but readonly externally, to limit the API surface.

`getInputs` could keep that option open by returning a copy, `{ ...this.inputs }`, but tbh this is pretty minor if we're disciplined when we write other code.